### DISCUSSION TITLE
Can we turn off a pump without turning off the pool?

### DISCUSSION THREAD

**Discussion by Crewski** [tagyoureit/nodejs-poolController#469] [General]
I have a Hayward VS pump communicating over rs485 now.  I setup a circuit for "Max Speed" (type Master Cleaner, have also tried it with Generic) that I use to put my pump on max speed.  I really would only use this speed to maximize the skimmer for an hour or so, and for backwashing.  Ideally I would be able to turn the pump on and off for the backwash-rinse-filter cycle on my sand filter.  I'm seeing a couple issues with how I have this setup.

1.  Turning the pool off then on does not reset my Aquarite T9 SWG back to the setpoint, it stays at 0%.
2. When I turn the pool back on, the "Max Speed" circuit remains off (which I agree is how it should be) so I have to turn that back on.

So right now for backwashing I have to:
-Turn off pool
-Rotate filter to Backwash
-Turn on pool
-Turn on Max Speed circuit
-Backwash
-Turn off pool
-Rotate filter to Rinse
-Turn on pool
-Turn on Max Speed
-Rinse
-Turn off pool
-Rotate filter to Filter
-Turn on pool
-Hopefully remember to reset SWG output

So I would think being able to just turn the pump on/off would be great, unless you have a better way to go about this.

**Comment by rstrouse** [comment]
We appear to have a bug if the Aquarite is not retaining the settings.  However, it should not come back on unless the filter circuit is re-engaged.   Burn me a replay where:
1. Only the pool is on with the Aquaraite set properly.  
2. Then turn off the pool.
3. Wait about 30 seconds for njsPC to evaluate all the states
4. Turn the pool back on

For the backwash and rinse simply turning on the Max Speed circuit should make the pump start running at 3450 even without the pool circuit being on.

**Reply by Crewski** [reply]
Attached is the replay.  Aquarite starts at 30%, goes to 0%, then stays at 0% when the pool is turned back on.  I assume this is related to the bug I posted about the super chlorinate not going back to initial output.
[replay (3).zip](https://github.com/tagyoureit/nodejs-poolController/files/8806591/replay.3.zip)

Thanks for the tip on the Max Speed circuit.  I set it as type "generic" and was able to toggle it on/off with the pool off and have the pump react properly.

**Comment by Crewski** [comment]
Never mind, there isn't a bug.  The problem was on my end with syncing the setpoint in home assistant.  Thanks for helping.
